manifests/h/HERE/OLPCLI/NATIVE/16.3.5/HERE.OLPCLI.NATIVE.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI.NATIVE
+PackageVersion: 16.3.5
+InstallerType: portable
+Commands:
+- olp

```

Comment:
```suggestion
- olp
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Thanks @stephengillie!

I did not see this issue on my local machine. Maybe because I have already installed Microsoft Visual Studio.